.github/workflows/create-release.yml

```diff
@@ -0,0 +1,91 @@
+name: Create Release
+
+on: 
+  schedule:
+    - cron: '0 0 * * 3'
+  workflow_dispatch:
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.HCP_SDK_PIPELINE_TOKEN }}
```

Comment:
Would this work if we just gave the job `Write` permissions instead, instead of using the service acct token? https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs